resources/estimate_copy_number.R

```diff
@@ -148,4 +148,4 @@ run_ace_pipeline <- function(object, QDNAseqobjectsample, filename, ploidy){
   generate_CN_dfs(object, QDNAseqobjectsample, filename)
 }
 
-run_ace_pipeline(copyNumbersCalled, QDNAseqobjectsample, filename)
+run_ace_pipeline(copyNumbersCalled, QDNAseqobjectsample, filename, ploidy)
```

Comment:
Let's have a look at the results together on how adding this value changed the results.

Reply:
I understand this comment, although this is not directly related to the code